desktop-app/app/main.dev.js

```diff
@@ -196,8 +199,9 @@ const createWindow = async () => {
     if (urlToOpen) {
       openUrl(urlToOpen);
       urlToOpen = null;
+    } else {
+      mainWindow.show();
```

Comment:
The reason why I add `else` is that there is `mainWindow.show();` in `openUrl()` and two `mainWindow.show();` calls is going throw an error on my machine.